constants/action.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package constants
+
+// Build and repo events.
+const (
+	// ActionOpened defines the action for opening pull requests.
+	ActionOpened = "opened"
+
+	// ActionEdited defines the action for the editing of pull requests.
+	ActionEdited = "edited"
+
+	// ActionSynchronized defines the action for the synchronizing of pull requests.
+	ActionSynchronized = "synchronized"
```

Comment:
Does GitHub generate a webhook when the pull request is closed? If so, can we include that too?